adhocracy4/dashboard/forms.py

```diff
@@ -121,6 +128,19 @@ class Meta:
         fields = ["result"]
         required_for_project_publish = []
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields["result"].help_text = _(
+            "The results description will be shown in the result-tab. "
+            "Please describe a participation promise beforehand "
+            "(what will happen with the outcome?) and inform afterwards "
+            "about the outcome. If you add an image, please provide "
+            "an alternate text. It serves as a textual description "
+            "of the image content and is read out by screen readers. "
+            "Describe the image in approx. 80 characters. "
+            "Example: A busy square with people in summer."
+        )
+
```

Comment:
There is a `help_texts` dict you can add in Meta:
https://docs.djangoproject.com/en/4.2/topics/forms/modelforms/#overriding-the-default-fields

and don't forget to remove the migrations :)

Reply:
the migrations are there to remove the old help_text on the model, otherwise I feel it will be confusing having old one on the model and a new one on the form